app/src/main/java/bankingapp/App.java

```diff
@@ -10,13 +18,14 @@
 
 public class App {
 
+    static Scanner input = new Scanner(System.in);
```

Comment:
Okay. I see you've got two PRs with the same commit. I've had only one case breaking up a series of commits like this made sense. If we were working together daily, I'd expect you to give reviewers a heads-up in the PR description, and ask them to review the other PR first, or just make one PR for all the changes.

However, I understand why you did that, since I haven't committed to review your PRs daily. :-) I can work around it by reviewing only the parts which are new in git and commenting here. Not every reviewer will have that skill, though!